core/templates/dev/head/components/exploration_summary_tile_directive.html

```diff
@@ -1,9 +1,9 @@
 <script type="text/ng-template" id="summaryTile/exploration">
-  <md-card class="oppia-exploration-summary-tile" ng-class="{'small-width': !isWindowLarge}">
+  <md-card class="oppia-gallery-summary-tile" ng-class="{'small-width': !isWindowLarge}">
```

Comment:
Should be oppia-activity-summary-tile, and let's keep the styles specific to this directive in this file. (We're moving away from having oppia.css contain everything.)